Libraries/LibWeb/Loader/Resource.cpp

```diff
@@ -83,6 +83,8 @@ static String guess_mime_type_based_on_filename(const URL& url)
         return "image/png";
     if (url.path().ends_with(".gif"))
         return "image/gif";
+    if (url.path().ends_with(".jpg"))
```

Comment:
should this also check for .jpeg ?

Reply:
That's probably a good idea. Personally I have never come across an image with .jpeg or .JPEG extension, but I believe there are servers and image manipulation programs allowing .jpeg in their workflow.